docs/source/standard_arguments.rst

```diff
@@ -132,32 +135,33 @@ Expectations with `mostly` return exception lists even if they succeed:
     {
       "success": true
       "summary_obj": {
-        "exception_percent": 0.2, 
-        "partial_exception_index_list": [
+        "unexpected_percent": 0.2,
+        "partial_unexpected_index_list": [
           8,
           9
-        ], 
-        "partial_exception_list": [
-          8, 
+        ],
+        "partial_unexpected_list": [
+          8,
           9
-        ], 
-        "exception_percent_nonmissing": 0.2, 
-        "exception_count": 2
+        ],
+        "unexpected_percent_nonmissing": 0.2,
+        "unexpected_count": 2
       }
     }
 
+
 DataSet defaults
 ------------------------------------------------------------------------------
 
-This default behavior for `output_format`, `include_config`, `catch_exceptions` can be overridden at the DataSet level:
+This default behavior for `result_format`, `include_config`, `catch_exceptions` can be overridden at the DataSet level:
 
 .. code-block:: bash
 
-    my_dataset.set_default_expectation_argument("output_format", "SUMMARY")
+    my_dataset.set_default_expectation_argument("result_format", "SUMMARY")
```

Comment:
Does this mean that you decided not to treat `result_format` as a dict? (To allow for things like `max_partial_list_length`)

Reply:
No...a dict is still acceptable, but so is a string as a way to provide a simpler interface for such circumstances. Current documentation now shows both usages, and I have added a test where both properties are changed via the dict.